.buddy/ci.yml

```diff
@@ -0,0 +1,29 @@
+- pipeline: 'CI'
+  on: 'EVENT'
+  events:
+    - type: 'PUSH'
+      refs:
+        - 'refs/heads/*'
+  priority: 'NORMAL'
+  fail_on_prepare_env_warning: true
+  variables:
+    - key: 'CODECOV_TOKEN'
+      value: 'secure!98lZjvc3CKurbTtXVlQH16aV+fYmfEppicIF3doKpeL2eYhd9snLO+0n+srjBlL+.EciZ7pJQwo9GJwIabNI6dw=='
+      type: 'VAR'
+      encrypted: true
+  actions:
+    - action: 'Build dependencies'
+      type: 'BUILD'
+      region: 'us-west-2'
+      docker_image_name: 'smartcar/base'
+      docker_image_tag: '14'
+      execute_commands:
+        - 'npm ci --ignore-scripts'
+        - 'npm rebuild'
+        - 'npm run cover'
+        - 'bash <(curl -s https://codecov.io/bash)'
```

Comment:
can you convert this to following method : 
https://github.com/smartcar/api/pull/3648/files#diff-f50e7b20c498e63a6bc93450cf6c35d82d1b8a5007f80a36839df5129b93cd31R56-R58